packages/jest-cli/src/TestScheduler.js

```diff
@@ -289,8 +289,14 @@ export default class TestScheduler {
   _setupDefaultReporters(collectCoverage: boolean) {
     this.addReporter(
       this._globalConfig.verbose
-        ? new VerboseReporter(this._globalConfig)
-        : new DefaultReporter(this._globalConfig),
+        ? new VerboseReporter(
+            this._globalConfig,
+            this._getReporterOption('verbose'),
+          )
+        : new DefaultReporter(
+            this._globalConfig,
+            this._getReporterOption('default'),
+          ),
```

Comment:
SummaryReporter (line 304) will still print diffs when there will be more than 20 test suites.